tests/src/ClientTest.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
Let's move this test under the api folder, the test folders need to reflect the src folders and files.

Reply:
This is not testing a specific API but the client, which is the gateway to all APIs.